LUPAINIANS/data/map systems.txt

```diff
@@ -0,0 +1,119 @@
+system "RINGWORLD_SYSTEM" #placeholder name
+	pos 10000 10000 #just out of the way for now, can be changed later
+	government Quarg
+	attributes "ringworld"
+	arrival 700
+	departure 1000
+	habitable 700
+	belt 1245
+	fleet Quarg 610
```

Comment:
Most ringworlds have `fleet "Large Quarg"` as well as `fleet Quarg`, so it might make sense to add that here too.